doc/CAPI.rst.txt

```diff
@@ -274,19 +274,23 @@ array, or part of some larger record array.  But, they may have other uses...
 ``NPY_WRITEABLE``
     True only if the data buffer can be "written" to.
 
-``NPY_UPDATEIFCOPY``
+``NPY_WRITEBACKIFCOPY``
     This is a special flag that is set if this array represents a copy
     made because a user required certain flags in ``PyArray_FromAny`` and
     a copy had to be made of some other array (and the user asked for
     this flag to be set in such a situation). The base attribute then
     points to the "misbehaved" array (which is set read_only). When
-    the array with this flag set is deallocated, it will copy its
+    ``PyArray_ResoveWritebackIfCopy`` is called, it will copy its
```

Comment:
missing `l` in `Resolve`.

Text should probably say something like "if you use this flag, you have to later call `PyArray_ResolveWriteBackIfCopy` on the array. If the the array has the `NPY_WRITEBACKIFCOPY` flag set, it will copy its contents back ... Otherwise, this operation is a no-op."